tests/test_demographics.py

```diff
@@ -114,47 +112,47 @@ def test_demographicsPUT(self):
                }
 
         self.login()
-        rv = self.client.put('/api/demographics/%s' % TEST_USER_ID,
+        response = self.client.put('/api/demographics/%s' % TEST_USER_ID,
                 content_type='application/json',
                 data=json.dumps(data))
 
-        self.assert200(rv)
-        fhir = json.loads(rv.data)
+        assert response.status_code == 200
+        fhir = response.json
         for item in fhir['telecom']:
             if item['system'] == 'phone':
                 if item['use'] == 'home':
-                    self.assertEqual(alt_phone, item['value'])
+                    assert alt_phone == item['value']
                 elif item['use'] == 'mobile':
-                    self.assertEqual(phone, item['value'])
+                    assert phone == item['value']
                 else:
                     self.fail(
                         'unexpected telecom use: {}'.format(item['use']))
             else:
                 self.fail(
                     'unexpected telecom system: {}'.format(item['system']))
-        self.assertEqual(fhir['birthDate'], dob)
-        self.assertEqual(fhir['deceasedDateTime'], dod)
-        self.assertEqual(fhir['gender'], gender.lower())
-        self.assertEqual(fhir['name']['family'], family)
-        self.assertEqual(fhir['name']['given'], given)
+        assert fhir['birthDate'] == dob
+        assert fhir['deceasedDateTime'] == dod
+        assert fhir['gender'] == gender.lower()
+        assert fhir['name']['family'] == family
+        assert fhir['name']['given'] == given
         # ignore added timezone and empty extensions
-        self.assertEqual(2, len(
-            [ext for ext in fhir['extension']
-             if 'valueCodeableConcept' in ext]))
-        self.assertEqual(3, len(fhir['careProvider']))
-        self.assertTrue(Reference.practitioner(pract_id).as_fhir() in fhir['careProvider'])
+        assert 2 == len([ext for ext in fhir['extension']
```

Comment:
I might switch the assertion around next time to the more customary form (so the value you're checking against is on the right) if you're already changing the line

No need to change this now though
```
assert len([ext for ext in fhir['extension'] == 2
```

Reply:
Gotcha